post-commit

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+# by Fabio William Conceição
```

Comment:
Why do we need a post commit? This can reside only in your own git I think

Reply:
This is only a template as a reference for users on how to do it, some people don't know to do it and don't know they're examples inside the folder .git/hooks.

Reply:
I do not think we need it here. They can have examples on other websites. :)

Reply:
Ok, I'll gonna remove it for us.